package.json

```diff
@@ -17,6 +20,14 @@
     "url": "https://github.com/OpenPaaS-Suite/openpaas-auth-client/issues"
   },
   "homepage": "https://github.com/OpenPaaS-Suite/openpaas-auth-client#readme",
+  "dependencies": {
+    "@babel/core": "^7.10.5",
+    "@babel/preset-env": "^7.10.4",
+    "babel-loader": "^8.1.0",
+    "oidc-client": "^1.10.1",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
```

Comment:
`babel*` & `webpack*` should be `devDependencies`?

Reply:
moved, will see if install works well in this case